src/EnergyPlus/Furnaces.cc

```diff
@@ -9138,10 +9066,7 @@ namespace Furnaces {
 
         //  OnOffAirFlowRatio = Par(8)
         if (state.dataFurnaces->Furnace(FurnaceNum).FurnaceType_Num == DataHVACGlobals::UnitarySys_HeatPump_WaterToAir) {
-            bool errFlag = false;
-            HeatPumpRunFrac(state, FurnaceNum, PartLoadRatio, errFlag, RuntimeFrac);
```

Comment:
Parent objects do not need to calculate runtime fraction. It's now handled by the child coils directly.